.github/workflows/ubuntu.yml

```diff
@@ -37,65 +37,75 @@ jobs:
     steps:
     - uses: actions/checkout@v4
 
-    - name: Cache dependencies
-      uses: actions/cache@v3
-      with:
-        path: |
-          /usr/local/lib
-          /usr/local/include
-          ~/.cache
-        key: ${{ runner.os }}-build-cache-${{ hashFiles('**/CMakeLists.txt') }}
-        restore-keys: |
-          ${{ runner.os }}-build-cache-
+#    - name: Cache dependencies
+#      uses: actions/cache@v3
+#      with:
+#        path: |
+#          /usr/local/lib
+#          /usr/local/include
+#          ~/.cache
+#        key: ${{ runner.os }}-build-cache-${{ hashFiles('**/CMakeLists.txt') }}
+#        restore-keys: |
+#          ${{ runner.os }}-build-cache-
 
     - name: Add Eigen3
-      run: sudo apt install libeigen3-dev
-
-    - name: Add TinyXML2
       run: |
-        sudo apt-get update
-        sudo apt-get install libtinyxml2-dev
-
-    - name: Add urdfom_headers
+        sudo apt update
+        sudo apt install libeigen3-dev
+        
+    - name: Add Rust (Required by Clarabel)
       run: |
-        sudo apt-get update
-        sudo apt-get install -y liburdfdom-headers-dev liburdfdom-tools
+        curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+        echo 'export PATH="$HOME/.cargo/bin:$PATH"' >> ~/.bashrc
+        source ~/.bashrc
+        echo "::add-path::$HOME/.cargo/bin"
```

Comment:
this command doesn't seem to be working

Reply:
This seems to be a GitHub issue, I'll try pushing again after resolving the other problems.